klog.go

```diff
@@ -0,0 +1,86 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
can we move klog.go in a package `pkg/klog` for example?

Reply:
Done: https://github.com/DataDog/datadog-operator/pull/192/commits/48d7bf11a9320226e775fe8fbb983697e88c5390
Thanks for the suggestion.